e2e/wdio/core/pages/input-group.po.ts

```diff
@@ -0,0 +1,34 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class InputGroupPo extends CoreBaseComponentPo {
+    url = '/inputGroup';
+    root = '#page-content';
+    contentPage = '#page-content';
+
+    inputFields = '.fd-input-group__input';
+    inputGroupSearchText = 'fd-input-group~small';
+    inputButtons = 'component-example .fd-input-group__button:not(.is-disabled)';
+    playgroundCheckbox = '.fd-checkbox__label';
+    playgroundInputField = '.fd-input.form-control';
+    rightTextAddon = '.fd-playground__content span';
+    playgroundInputButton = '.fd-playground__content button';
+
+    open(): void {
+        super.open(this.url);
+        waitForElDisplayed(this.root);
+        waitForPresent(this.contentPage);
```

Comment:
maybe use this.title  (it's saved on CoreBaseComponentPo) - will wait for page title to be present